osu.Game.Rulesets.Catch/Difficulty/CatchDifficultyCalculator.cs

```diff
@@ -38,11 +38,13 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
 
             return new CatchDifficultyAttributes
             {
+                DirectionChangeCount = ((Movement)skills[0]).DirectionChangeCount,
                 StarRating = Math.Sqrt(skills[0].DifficultyValue()) * star_scaling_factor,
                 Mods = mods,
                 ApproachRate = preempt > 1200.0 ? -(preempt - 1800.0) / 120.0 : -(preempt - 1200.0) / 150.0 + 5.0,
                 MaxCombo = beatmap.HitObjects.Count(h => h is Fruit) + beatmap.HitObjects.OfType<JuiceStream>().SelectMany(j => j.NestedHitObjects).Count(h => !(h is TinyDroplet)),
-                Skills = skills
+                Skills = skills,
+                CircleSize = beatmap.BeatmapInfo.BaseDifficulty.CircleSize
```

Comment:
Are you missing some changes? This is writing circle size to the attributes, but the property doesn't exist (see compile errors), and neither does usage exist in the performance calculator.

Reply:
I used it for testing at some point and I forgot to remove it.